app/Main.hs

```diff
@@ -80,13 +81,21 @@ parseOptions :: Parser LintOptions
 parseOptions =
     LintOptions <$> -- CLI options parser definition
     version <*>
+    configFile <*>
     outputFormat <*>
     ignoreList <*>
     files <*>
     parseRulesConfig
   where
     version = switch (long "version" <> short 'v' <> help "Show version")
     --
+    -- | Parse the config filename to use
+    configFile =
+        optional
+            (strOption
+                 (long "config" <> short 'c' <> metavar "FILENAME" <>
+                  help "Use a custom file as configuration"))
```

Comment:
Would you mind to change it to `Path to the configuration file`?

Reply:
It is not just the path to, it needs to include the file name

Reply:
fair enough